developer/building/qubes-builder-v2.md

```diff
@@ -0,0 +1,143 @@
+---
+lang: en
+layout: doc
+permalink: /doc/qubes-builder-v2/
+redirect_from:
+- /en/doc/qubes-builder-v2/
+- /doc/QubesBuilder2/
+- /wiki/QubesBuilder2/
+ref: 64
+title: Qubes builder
+---
+
+This is a brief introduction of using Qubes Builder v2 to work with Qubes OS 
+sources. It will walk you through installing and configuring Builder v2 and 
+using it to fetch and build Qubes OS packages.
+
+For details and customization, use [Qubes OS v2 builder documentation]
+(https://github.com/QubesOS/qubes-builderv2/).
+
+# Overview
+
+In the second generation of Qubes OS builder, container or disposable qube 
+isolation is used to perform every stage of the build and release process. 
+From fetching sources to building, everything is executed inside an isolated 
+*cage* (either a disposable or a container) using an *executor*. For every 
+command that needs to perform an action on sources, like cloning and 
+verifying Git repos, rendering a SPEC file, generating SRPM or Debian 
+source packages, a new cage is used. Only the signing, publishing, and 
+uploading stages are executed locally outside a cage. 
+
+
+# Setup
+
+This is a simple setup for using a docker executor (a good default choice, 
+if you don't know which executor to use, use docker)
+
+1. First, clone the qubes-builder v2 repository into a location of your choice:
+    ```shell
+    git clone https://github.com/QubesOS/qubes-builderv2
+    cd qubes-builderv2/
+    ```
+
+2. Installing dependencies
```

Comment:
This (and the usermod below) needs to be done in the template for the chosen AppVM (if that's on qubes). Or in a StandaloneVM.

Reply:
chosen as in the one we are installing things in?